osgpkitools/ConnectAPI.py

```diff
@@ -52,17 +52,17 @@ def request_unauthenticated(self, config, name, email, phone, csr, vo='', cc_lis
         params = urllib.urlencode(params_list)
         headers = {'Content-type': config['content_type'],
                    'User-Agent': USER_AGENT}
-        conn = httplib.HTTPConnection(config['host'])
 
-        response = self.do_connect(conn, 'POST', config['requrl'], params, headers)
-        data = response.read()
-        check_failed_response(data)
-        conn.close()
+        # TODO: Remove this line when we get ports out of the ini configuration
+        host_default_port = config['host'].split(':')[0]
```

Comment:
What's in `config['host']`? If it's `ADDRESS:PORT`, then you want `[1]` instead of `[0]`.
Unless `host_default_port` is not supposed to be a port number but something else, in which case that variable should be renamed.
Also you're using `config['host']` here but you changed some uses below to `config['hostsec']`. Is that OK?

Reply:
Just a bad variable name, fixed. `host` and `hostsec` are the same except for the port specification (`80` vs `443` for the default configuration (and the few people that use this tool don't mess with the default config).